CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 option(QT_DEFAULT_MAJOR_VERSION "Major version of Qt to use (6 or 5)" 6)
 
+# Ensure we always export all symbols for Windows
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Eh, this is hacky. If we want to use it, we should set the property on our CMake targets instead of globally.

Reply:
It doesn't seem to help or change anything anyway